src/scss/_config.scss

```diff
@@ -200,8 +200,11 @@ $gorko-config: (
   ),
   'breakpoints': (
     'xs': '(max-width: 25em)',
+    'max-md': '(max-width: 50em)',
```

Comment:
You should be able to structure your css to use existing breakpoints (they are written this way to favos mobile-first design). Mixing min-width and max-width leads to rather unreadable and unreliable css. If you find the use of it absolutely necessary, please move the reused value (50em etc) to a variable.